source/_patterns/02-organisms/content-headers/content-header-profile.mustache

```diff
@@ -0,0 +1,28 @@
+<header class="content-header-profile">
+
+  <h1 class="content-header-profile__displayName">{{{displayName}}}</h1>
+
+  {{#details}}
+  <div class="content-header-profile__details">
+
+    <div>
+      {{#affiliations}}<span class="content-header-profile__affiliation">{{{.}}}</span>{{/affiliations}}
+    </div>
+
+    {{#emailAddress}}
+    <div class="content-header-profile__email">{{{emailAddress}}}</div>
```

Comment:
Is the email address meant to be clickable?

Reply:
No.

Reply:
Ok, should probably just be `{{`?